reindexer/terraform/service_reindex_request_creator.tf

```diff
@@ -1,5 +1,5 @@
 module "reindex_request_creator" {
-  source = "git::https://github.com/wellcometrust/terraform-modules.git//sqs_autoscaling_service?ref=v10.3.0"
+  source = "git::https://github.com/wellcometrust/terraform-modules.git//sqs_autoscaling_service?ref=v10.2.2"
```

Comment:
Why the step to 10.2.2?

Reply:
I can only guess, but I assume it's because 10.2.2 is the last version of the old modules supporting `env_vars_length` parameter. Probably the environment variables interpolation wasn't quite working

Reply:
Yes, it's because the  environment variables interpolation wasn't working.  I think the latest version has this removed, so I'll switch it back.

Reply:
After discussing this just now, I'm going to merge as is because we will revisit this when we move to the new infrastructure.